jeffjxu/Wesley-Central-Access-master
# edit client information when creating a new inquiry

able to edit a client's information while creating a new inquiry for that client

how to test:
- go to create new inquiry form for an existing client
- expand the client form and edit some information
- create the inquiry
- check if the client information was correctly updated
